prow/scripts/cluster-integration/kyma-aks-long-lasting.sh

```diff
@@ -207,7 +207,28 @@ function addGithubDexConnector() {
     export DEX_CALLBACK_URL="https://dex.${CLUSTER_NAME}.build.kyma-project.io/callback"
     go run "${KYMA_PROJECT_DIR}/test-infra/development/tools/cmd/enablegithubauth/main.go"
 }
+function getLetsEncryptCertificate {
+
+  printf "\nChecking if certificate is already in GCP Bucket."
+      mkdir -p ./letsencrypt/live/"${DOMAIN}"
+ if [[ $(gsutil ls gs://kyma-prow-secrets/nightly-gke-tls-integration-app-client-cert.encrypted) ]];
+ then
+    printf "\nCertificate/privatekey exists in vault. Downloading..."
+
+    cp /etc/credentials/sa-gke-kyma-integration/service-account.json letsencrypt
+  #copy the files
+    gsutil cp gs://kyma-prow-secrets/nightly-gke-tls-integration-app-client-cert.encrypted "./letsencrypt/live/${DOMAIN}" 
+    gsutil cp gs://kyma-prow-secrets/nightly-gke-tls-integration-app-client-key.encrypted "./letsencrypt/live/${DOMAIN}" 
+#decrypt key and cert
+printf "decrypting certs"
```

Comment:
Please fix formatting.

Reply:
was this not what the prow was insisting?